packages/ove-app-audio/src/index.js

```diff
@@ -79,21 +81,25 @@ app.get('/operation/:name(' + operationsList + ')', function (req, res) {
         }
         res.status(HttpStatus.OK).set(Constants.HTTP_HEADER_CONTENT_TYPE, Constants.HTTP_CONTENT_TYPE_JSON).send(
             JSON.stringify({ status: (isComplete ? Constants.BufferStatus.COMPLETE : Constants.BufferStatus.BUFFERING) }));
-
     } else {
         // other commands receive additional query parameters.
-        let message = { operation: { name: name, executionTime: (new Date().getTime() + Constants.OPERATION_SYNC_DELAY) } };
+        let message = {
+            operation: {
+                name: name,
+                executionTime: (new Date().getTime() + Constants.OPERATION_SYNC_DELAY)
+            }
+        };
         if (name === Constants.Operation.SEEK) {
             // We assume that the seek time is properly set instead of enforcing any strict type checks.
             message.operation.time = req.query.time;
-        } 
+        }
         if (name === Constants.Operation.PLAY) {
```

Comment:
Back to back if checks instead of 'else if', :). I don't think there ever can be a situation where the `name` is equal to both `seek` and `play`.

Reply:
changed, though it makes the code look nastier for negligible performance improvement.

Reply:
@davidbirchwork, the compiler would optimise the above. 

However, I think the use of `if-if-if` instead of `if-elseif-else` does seem to be something you have taught yourself because the latter is what most languages expect you to program with. I don't see anything that makes the code look nastier in doing that. Am I missing something here?

Similar to nested if statements a very long sequence of else-if would of course would lead to the IDEs/tools complaining about code complexity, having many non-nested if statements instead of a lengthy block fools them not to. However, that does not mean the code is better. The tools ideally expect you to implement it very differently such that you would significanly reduce the number of else-if blocks.

Reply:
@senakafdo this is simply a personal view in favour of code readability given we can rely on good compilers. if-else chains have created a 20line logically connected statement which must be considered as a whole rather than 3 much shorter independent ones which are easier to find, read and maintain.  I know these monster statements are normal in .js. I guess i'm just slow in adapting to them ;) 

I agree that really a switch statement is what's needed here beyond a certain scale or even better a decent form of pattern matching like in haskell ;-) 

anyway this is changed to support your style and i'll try and follow it in future :)